config/200-clusterrole.yaml

```diff
@@ -65,7 +65,7 @@ metadata:
 rules:
   - apiGroups: [""]
     resources: ["secrets"]
-    verbs: ["get", "list", "watch"]
+    verbs: ["get", "list", "watch", "update"]
```

Comment:
I don't think we should change the role to be able to update all secrets. Instead create another rule that can update the specific secret that we need

Reply:
`tekton-triggers-core-interceptors` cluster role is only used for `tekton-triggers-core-interceptors` deployment so i believe this is for specified one only

Let me know if this is your concern or something else

Reply:
Right the concern is that this allows the core interceptor server access to update any secret in the cluster which is a bit scary. What we should do is to make sure it can only update the TLS secret for the cluster interceptor.
See the example here where the webhook role can only update the webhook-certs secret using the `resourceName` field: https://github.com/tektoncd/pipeline/blob/main/config/200-role.yaml#L59-L65